deploy/charts/harvester/README.md

```diff
@@ -1,108 +1,58 @@
-# Harvester Helm Chart [Development]
-
-Harvester is an open source Hyper-Converged Infrastructure(HCI) solution based on Kubernetes.
-
-Note: the master branch is under active development, please use the `stable` branch if you need a stable version of the Harvester chart.
+# Harvester Helm Chart
 
 ## Chart Details
 
 This chart will do the following:
 
 - Deploy a KubeVirt Operator if needed, defaults to deploy.
-- Deploy a KubeVirt CRD resource to enable KubeVirt if needed, defaults to deploy.
 - Deploy the Harvester resources.
 - Deploy [Longhorn](https://longhorn.io) as the built-in storage management.
+- Deploy Containerized Data Importer (CDI) as supporting third-party storage.
+- Deploy csi-snapshotter
+- Deploy Snapshot Validation Webhook
+- Deploy whereabouts
 
-### Prerequisites
-
-- Kubernetes 1.16+.
-- Helm 3.2+.
-- [Multus CNI](https://github.com/k8snetworkplumbingwg/multus-cni). An [example chart](https://github.com/rancher/rke2-charts/tree/main-source/packages/rke2-multus/charts).
-- [Harvester CRDs](https://github.com/harvester/harvester/tree/master/deploy/charts/harvester-crd)
-
-### Installing the Chart
-
-To install the chart with the release name `harvester`.
-
-```bash
-## create target namespace
-$ kubectl create ns harvester-system
-
-## install chart to target namespace
-$ helm install harvester harvester --namespace harvester-system --set service.harvester.type=NodePort
```

Comment:
Does that mean these `helm` commands no longer work? I haven't tried them yet.

Reply:
we've removed that part https://github.com/harvester/docs/pull/628